plone/app/dexterity/behaviors/tests/test_metadata.py

```diff
@@ -1,4 +1,9 @@
 # -*- coding: utf-8 -*-
+from zope.schema.interfaces import IVocabularyFactory
+from plone.app.dexterity.behaviors.metadata import ICategorization
+from zope.component import getUtility
+from plone.registry.interfaces import IRegistry
+from plone.app.dexterity.testing import DEXTERITY_INTEGRATION_TESTING
```

Comment:
This test blows up the imports quite a bit.
I could move the test a level up, into plone.app.dexterity.tests instead plone.app.dexterity.behaviors.tests. The other also has does plone integration test-layered tests.
but on the other hand I like the grouping of tests together more...